src/Symfony/Component/DependencyInjection/ContainerInterface.php

```diff
@@ -21,7 +22,7 @@
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
  */
-interface ContainerInterface
+interface ContainerInterface extends PsrContainerInterface
```

Comment:
Hmm… seems like it would indeed be a BC-break so… only for final classes I guess. https://3v4l.org/I8jO2